services/src/main/java/com/chatbot/services/pubsubservices/PubSubAuth.java

```diff
@@ -0,0 +1,32 @@
+package com.chatbot.services.pubsubservices;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Collections;
+import java.util.Map;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class PubSubAuth {
+  public void verifyRequest(final Map<String, String> headers)
+      throws GeneralSecurityException, IOException {
+    final GoogleIdTokenVerifier verifier =
+    new GoogleIdTokenVerifier.Builder(new NetHttpTransport(),
+    new JacksonFactory()).setAudience(Collections.singletonList(System.getenv("pubsubAudience")))
+        .build();
```

Comment:
Indentation looks bad at some places. Can you run it over files and fix it?

Reply:
updated